client/src/controllers/SwapController.ts

```diff
@@ -46,6 +46,7 @@ export class SwapController extends Controller<
     icon: { default: '', type: String },
     loading: { default: false, type: Boolean },
     src: { default: '', type: String },
+    reflect: { default: false, type: Boolean },
```

Comment:
Nitpick but can we keep these in alpha order?

So `reflect` would be before `src`.